app/services/true_layer/importers/import_account_holders_service.rb

```diff
@@ -0,0 +1,36 @@
+module TrueLayer
+  module Importers
+    class ImportAccountHoldersService
+      def initialize(api_client, bank_provider)
+        @api_client = api_client
+        @bank_provider = bank_provider
+      end
+
+      def call
+        bank_provider.bank_account_holders.clear
+        bank_provider.bank_account_holders.create!(mapped_resources)
+      end
+
+      private
+
+      attr_reader :api_client, :bank_provider
+
+      def mapped_resources
+        true_layer_resources.map do |resource|
+          {
+            true_layer_response: resource,
+            full_name: resource[:full_name],
+            full_address: resource[:addresses]&.map do |address|
```

Comment:
💭 If we are storing **multiple** addresses in this field, would it be better just to call it `addresses` to be clear as what the fields has? (`full_address` seems a bit deceiving).  

Also, I'm not entirely familiar with the format of the returned addresses, but is there any chance that any of the `address.values` already has commas on it? (if so, I would consider storing these addresses in a more structured way, serialised).